### PR TITLE
Feature/436475 parameter maintenance remove total from default parameter page

### PR DESCRIPTION
**Pull request by rekhadhankhar2018**
Remove TOTAL from Default Parameter page

### CODE REVIEW

**Review by johnbenjamin-defra — DISMISSED** [review]

File: src/EPR.Calculator.Frontend/Controllers/DefaultParametersController.cs [hunk 1]
```diff
@@ -6,72 +6,92 @@
...
             try
             {
-                var parameterSettingsApi = _configuration.GetSection("ParameterSettings").GetSection("DefaultParameterSettingsApi").Value;
+                var parameterSettingsApi = this.configuration.GetSection("ParameterSettings").GetSection("DefaultParameterSettingsApi").Value;
 
-                var client = _clientFactory.CreateClient();
+                var client = this.clientFactory.CreateClient();
                 client.BaseAddress = new Uri(parameterSettingsApi);
-                var year = _configuration.GetSection("ParameterSettings").GetSection("ParameterYear").Value;
+                var year = this.configuration.GetSection("ParameterSettings").GetSection("ParameterYear").Value;
```
[the remark this answers is not in the document]
Comment (rekhadhankhar2018) [reply src/EPR.Calculator.Frontend/Controllers/DefaultParametersController.cs:49]: this code is already replaced in my previous PR Uday

[the remark this answers is not in the document]
Comment (rekhadhankhar2018) [reply src/EPR.Calculator.Frontend/Controllers/DefaultParametersController.cs:49]: thanks Uday

[the remark this answers is not in the document]
Comment (rekhadhankhar2018) [reply src/EPR.Calculator.Frontend/Controllers/DefaultParametersController.cs]: done

File: src/EPR.Calculator.Frontend/Controllers/DefaultParametersController.cs [hunk 2]
```diff
@@ -6,72 +6,92 @@
...
+                        this.ViewBag.PreparationCosts = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.LocalAuthorityDataPreparationCosts);
+                        this.ViewBag.SchemeSetupCosts = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.SchemeSetupCosts);
+                        this.ViewBag.LateReportingTonnage = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.LateReportingTonnage);
+                        this.ViewBag.MaterialityThreshold = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.MaterialityThreshold);
+                        this.ViewBag.BadDebtProvision = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.BadDebtProvision);
+                        this.ViewBag.Levy = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.Levy);
+                        this.ViewBag.TonnageChange = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.TonnageChangeThreshold);
+                        this.ViewBag.EffectiveFrom = defaultSchemeParameters[0].EffectiveFrom;
```
[the remark this answers is not in the document]
Comment (rekhadhankhar2018) [reply src/EPR.Calculator.Frontend/Controllers/DefaultParametersController.cs]: done that

**Review by johnbenjamin-defra — APPROVED** [review]

**Pull request merged**